test/hooks.js

```diff
@@ -129,6 +133,33 @@ test('catches afterResponse errors', async t => {
 	}), errorString);
 });
 
+test('init', async t => {
```

Comment:
Can this be more descriptive? Maybe something like 'init is called with options'?

Reply:
Of course :D I'll rename other hook tests too.